src/support_sphere/lib/logic/cubit/login_state.dart

```diff
@@ -14,17 +15,19 @@ final class LoginState extends Equatable {
   final String password;
   final FormzSubmissionStatus status;
   final bool isValid;
+  final bool isAllFieldsFilled;
   final bool showPassword;
   final String? errorMessage;
 
```

Comment:
```suggestion

  bool get isLoginButtonEnabled => isValid && isAllFieldsFilled;

```

You can have a computed property like so for `isLoginButtonEnabled`.

Reply:
updated.